cvat/apps/engine/cache.py

```diff
@@ -321,15 +321,14 @@ def _read_raw_images(
                 cloud_storage_instance.bulk_download_to_dir(
                     files=files_to_download, upload_dir=tmp_dir
                 )
-                media = preload_images(media)
+                media = load_images(images=media, preload=False)
 
-                for checksum, (_, fs_filename, _) in zip(checksums, media):
-                    if checksum and not md5_hash(fs_filename) == checksum:
+                for checksum, media_item in zip(checksums, media):
+                    if checksum and not md5_hash(media_item[1]) == checksum:
                         slogger.cloud_storage[db_cloud_storage.id].warning(
                             "Hash sums of files {} do not match".format(file_name)
                         )
-
-                yield from media
+                    yield media_item
```

Comment:
I think you could replace this with `yield load_image(media_item)`; then the `load_images` call won't be needed at all.

Reply:
done